src/Compilers/Core/Portable/Diagnostic/CommonMessageProvider.cs

```diff
@@ -243,6 +243,7 @@ public string GetIdForErrorCode(int errorCode)
         public abstract int ERR_ModuleEmitFailure { get; }
         public abstract int ERR_EncUpdateFailedMissingAttribute { get; }
         public abstract int ERR_InvalidDebugInfo { get; }
+        public virtual int? ERR_FunctionPointerTypesInAttributeNotSupported => null;
```

Comment:
>null [](http://example.com/codeflow?start=79&length=4)

I think for VB we should have an error code too. Just in case. Perhaps we can use ERR_TooLongOrComplexExpression or some other suitable existing code.